hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/PrngSystemContract.java

```diff
@@ -102,7 +102,7 @@ public FullResult computeFully(@NonNull final Bytes input, @NonNull final Messag
                     PrecompiledContract.PrecompileContractResult.halt(Bytes.EMPTY, Optional.of(INVALID_OPERATION)),
                     gasRequirement,
                     null);
-        } catch (NullPointerException e) {
+        } catch (Exception e) {
```

Comment:
Not in the scope, but in the current flow we handle only two types of exceptions. If we have something throw another  exception type, we are going to leak it out. 
Putting the most common type at the end of the `catch` chain, we ensure that all exceptions are handled.

Discussed with the team. 
Any comments?